ui/components/PageStatus.tsx

```diff
@@ -23,7 +23,9 @@ function PageStatus({ conditions, suspended, className }: StatusProps) {
           suspended
             ? IconType.SuspendedIcon
             : ok
-            ? IconType.CheckCircleIcon
+            ? ok === ReadyType.Reconciling
+              ? IconType.ReconcileIcon
+              : IconType.CheckCircleIcon
             : IconType.FailedIcon
```

Comment:
Got lost in three ternaries here.

Would it be possible to

- assign the calculated the icon type to a constant before the return statement
- simplify the multiple ternaries or replace them with if/elses for readability
- just pass the final calculated value as the type prop instead of the whole complex if/else statement?